### DISCUSSION TITLE
How do we define ice sheet vs periphery?

### DISCUSSION THREAD

**Discussion by willkochtitzky** [IACS-cryo/Delineation-WG#24] [General]
I agree with Mathieu Morlighem @mmorligh that this is the first most important question (Mathieu, please correct me if I mischaracterized your comment) and then all of our work will flow from here if we can agree on this definition. Could people post what this definition should be? Ideally referenced from the literature?

I can start to add that [Cogley et al (2011)](https://wgms.ch/downloads/Cogley_etal_2011.pdf) define ice sheet as: "An ice body that covers an area of continental size, generally defined as covering 50 000 km2 or more."

**Comment by MathieuMorlighem** [comment]
Hi @willkochtitzky, yes I have done some research too and that's the definition that I find consistently:
- https://nsidc.org/learn/parts-cryosphere/ice-sheets/ice-sheet-quick-facts
- https://education.nationalgeographic.org/resource/ice-sheet/
- https://en.wikipedia.org/wiki/Ice_sheet
- https://www.britannica.com/science/ice-sheet
etc

If we apply this definition strictly, only ice bodies not connected to the ice sheets are considered glaciers. It is a simple definition that is very easy to apply in an objective way. I am sorry to repeat myself but I think it is important: it is not because we may decide use this strict definition that the RGI should stop including weakly or strongly connected glaciers in Greenland and Antarctica! (I am actually more and more convinced that the RGI should include the ice sheets for completeness ;) ).

**Comment by willkochtitzky** [comment]
I think perhaps the most popular(?) for Greenland has been [Rastner et al. (2012)](https://tc.copernicus.org/articles/6/1483/2012/tc-6-1483-2012.pdf):

– CL0: no connection;
– CL1: weak connection (clearly separable by drainage divides in the accumulation region, not connected or only in contact in the ablation region);
– CL2: strong connection (difficult to separate in the ac- cumulation region and/or confluent flow in the ablation region).

**Reply by eberthie** [reply]
A short addition to that. CL2 are generally (always?) excluded for studies looking at the Global (or RGI5-only) glacier mass balance. If useful for the group (and the first deliverable), I can make a short literature check to be sure this convention was followed by all, starting with Gardner et al. 2013 (and IPCC AR4).

**Reply by regine** [reply]
To my knowledge all glacier studies (at least those considered in IPCC for the past/future glacier changes) have used the same domain (RGI no and weak connectivity based on Rastner) so they are consistent.

Regarding what the RGI should include: it has been discussed to include the ice sheets, however, where ice sheet/glacier data are stored, seems secondary to the actual issue of what the ice sheet and glacier folks are using in their assessements/projections and possible redefinition of current RGI glacier domains.

While it may look east to just remove the weak connectivity glaciers from glacier assessments/projection, there is scientific reasons why they have been included in the global glacier studies, mostly that they "behave" more like glaciers and are often better modeled with "glacier" approaches given their typical higher sensitivities and small sizes.

**Reply by willkochtitzky** [reply]
@regine How would you define a glacier and an ice sheet and draw the line between them? Do you have a preferred definition or a proposal for one?

**Comment by MathieuMorlighem** [comment]
Thanks Regine! I appreciate your perspective and agree that some tools and methods may be more appropriate for a region or another, but tools and methods change all the time. I don't think we should use these as criteria to *define* what a glacier or an ice sheet is (just like ice caps "behave" like ice sheets but are not defined as ice sheets).

I am all for people continuing to do what they do, but if we want to avoid double counting, the easiest may be to have simple spatial masks (For example, and this is just an idea, RGI region 5 would have two subregions, 5.1 for the periphery of the ice sheet, including ice free land and periphery glaciers in 2020, and 5.2 for the ice sheet as defined in 2020, and people reporting on future mass loss would need to integrate their $\Delta$thickness on every mask and report the mass change in the right bucket)

**Reply by regine** [reply]
It´s not that easy. For DEM differencing based methods of past change one can post-process to any mask but things are, e.g., more complicated for modeling/projections. Not sure which domains you are referring to above ? What is peripheral ?
Will, I don´t have a concrete proposal to change the current domain for glacier work. This is what this WG is about but we first need a better understanding among each other why studies used what and why and what the problems are with using the current or another masks (e.g. back to my suggestion of compiling this as a basis for possible redefining domains).

**Reply by willkochtitzky** [reply]
Regine, do we have a current definition of what is an ice sheet and what is a periphery glacier that the community has agreed on? From the Cogley glossary that I linked to above I am only seeing a definition of ice sheet, but not periphery glacier. Has anyone defined what a periphery glacier is? 
The definition of ice sheet above is based on area, which then suggests this is a very simple activity of just looking at connected (I assume?) glacier area.

**Reply by regine** [reply]
Will, there is no definition in Cogley et a, but the glacier community has a clear definition through the RGI (a region called Greenland periphery and defined unambigously by defined outlines). This has alse been clarified even more by removing connectivity level 2 from RGI 7 based on what glacier folks have used in the past (just connectivity level 0 and 1)
see: https://www.glims.org/rgi_user_guide/regions/rgi05.html
"05: Greenland Periphery
Glaciers with connectivity level 2 [Rastner et al., 2012] have been removed entirely from the RGI 7.0, since they are typically not included in mass change assessments or projections of glaciers outside the ice sheet. This change removes a source of confusion when differentiating between glaciers and the ice sheet proper. See the “Version history” section below for a description of the connectivity levels and why they were included in previous versions."

**Reply by regine** [reply]
Will, just to follow up: The term Greenland periphery was "invented" by the RGI community. The idea was to find a name that makes clear that the corresponding RGI domain will not be confused with the all ice in Greenland and be as intuitive as possible to indicate where these ice masses are located, thus Greenland periphery for the sake of the RGI. Of course the term can be understood broadly in other ways. Same holds for region names like "Central Europe" or High Mountain Asia regions. There are all just RGI names following the same logic: geographically intuitive short names (although the terms are used in different ways in other context).

**Reply by willkochtitzky** [reply]
Thanks Regine! This is really useful context and history. Thanks for pointing to the RGI resources.
